CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to Sakai
 
-Contributions to Sakai from all comers are welcome and encouraged.
+Contributions to Sakai from all comers are very welcome and encouraged.
```

Comment:
This isn't https://jira.sakaiproject.org/browse/SAK-2907. That was closed 10 years ago.